main.js

```diff
@@ -0,0 +1,24 @@
+var menu = document.querySelector(".menu-mobile");
+var ham = document.querySelector(".ham");
```

Comment:
How about using `const` and `let`? It's actually a standard now since all the world is in awe of changes introduced by ES6 :D

Reply:
I've used "let", is it ok?